File: python/leetcode/binarysquare.py
class BinarySquare:
  @classmethod
  def Solver(n):
    i = 0
    odd = True
    while (i < n):
      j = 0
      while (j < n):
          if (odd):
              if (j % 2 == 0):
                  print(0, end='')
              else:
                  print(1, end='')
              j += 1
          else:
              j += 1
              if (j % 2 == 0):
                  print(0, end='')
              else:
                  print(1, end='')

        odd = not odd
        i += 1
        print()

File: go/vidio/binarysquare.go
package vidio

import (
	"bytes"
	"fmt"
)

func (BS *BinarySquare) Solver(n int) {
	for i := 0; i <= n; i++ {
		var b bytes.Buffer
		for j := 0; j <= n; j++ {

			if i%2 == 0 {
				if j%2 == 0 {
					b.WriteString("0")
				} else {
					b.WriteString("1")
				}
			} else {
				if j%2 == 0 {
					b.WriteString("1")
				} else {
					b.WriteString("0")
				}
			}
		}

		fmt.Println(b.String())
	}
}
